tutorials/introduction.md

```diff
@@ -1,7 +1,131 @@
 ---
 sidebar_position: 1
-sidebar_label: Introduction
-slug: /introduction
+sidebar_label: Token Gating Tutorial
+slug: /token-gating-tutorial
 ---
 
-# Introduction
\ No newline at end of file
+# Collab.Land Token Gating Tutorial
+## Overview
+This guide walks you through creating your own token gate for your website using the Collab.Land API. This enables you to restrict access to elements of or pages of your site based on ownership of certain blockchain assets.
+
+### Website Gating Logic Flow
+View logic diagram [on whimsical](https://whimsical.com/embed/S5jqcu5XzqBrnRtvgrPBj1@2Ux7TurymNLRX4obG1NB).
+
+This tutorial implements gating logic by calling Collab.Land API with a user's wallet address. The API checks against the website-defined ⚠ link needed! [Token-Gating Role] definitions and returns `True/False` if the address fits the provided criteria or not.
+
+⚠ Documentation needed 
+- json definition of TGRs and an example needed on [this page of docs](https://dev.collab.land/docs/local-development-setup/configuring-tgr)
```

Comment:
TGR example and schema needed on https://dev.collab.land/docs/local-development-setup/configuring-tgr